content/news/2023-01-24-gpu-jupyterlab-galaxy/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 'GPU-enabled JupyterLab in Galaxy for AI'
+date: '2023-01-24'
+tease: 'GPU-enabled JupyterLab in Galaxy for AI'
+hide_tease: true
+authors: 'Anup Kumar'
+tags: [tool]
+subsites: [all-eu]
+main_subsite: eu
+---
+
+
+### Introduction
```

Comment:
These Headings should be heading level 2 (e.g: `## Introduction`).
It's important to not skip heading levels, since they are used by some users to help navigate the site.
The "title" attribute in the header generates a level 1 heading on the news entry page, so the next heading level needs to be 2.

Read more [here](https://www.w3.org/WAI/tutorials/page-structure/headings/)

Reply:
Thanks I changed that.

Reply:
@ElectronicBlueberry @bgruening thanks!